testcases.py

```diff
@@ -266,6 +266,38 @@ def check(self):
         return True
 
 
+class TestCaseLongRTT(TestCaseHandshake):
+    @staticmethod
+    def abbreviation():
+        return "Y"
```

Comment:
Suggestions for a better abbreviation highly appreciated...

Reply:
How about "LR" ? I think we're getting to the point of not doing this well with abbreviations...

Reply:
Thanks.